$OpenBSD: patch-tools_virsh_c,v 1.1.1.1 2012/09/04 14:41:15 jasper Exp $

virsh.c:2192: warning: passing argument 1 of 'localtime' from incompatible pointer type

--- tools/virsh.c.orig	Tue Sep  4 15:03:14 2012
+++ tools/virsh.c	Tue Sep  4 15:06:01 2012
@@ -2179,6 +2179,7 @@ vshOutputLogFile(vshControl *ctl, int log_level, const
     const char *lvl = "";
     struct timeval stTimeval;
     struct tm *stTm;
+    time_t sec = stTimeval.tv_sec;
 
     if (ctl->log_fd == -1)
         return;
@@ -2189,7 +2190,7 @@ vshOutputLogFile(vshControl *ctl, int log_level, const
      * [YYYY.MM.DD HH:MM:SS SIGNATURE PID] LOG_LEVEL message
     */
     gettimeofday(&stTimeval, NULL);
-    stTm = localtime(&stTimeval.tv_sec);
+    stTm = localtime(&sec);
     virBufferAsprintf(&buf, "[%d.%02d.%02d %02d:%02d:%02d %s %d] ",
                       (1900 + stTm->tm_year),
                       (1 + stTm->tm_mon),
